packages/vector_graphics/lib/src/render_vector_graphics.dart

```diff
@@ -66,6 +66,21 @@ class RenderVectorGraphic extends RenderBox {
     markNeedsPaint();
   }
 
+  /// An additional ratio the picture will be transformed by.
+  ///
+  /// This value is used to ensure the computed raster does not
+  /// have extra pixelation from scaling.
```

Comment:
I'd find it helpful to have a little more explanation here about when/how that can happen, which way to drive the value and what trade offs that makes for memory/speed/fidelity.

Reply:
added more docs, but this is internal only for now. We could add a scale param on the widget itself if it is needed